src/shared/components/radar/radar.component.tsx

```diff
@@ -1,36 +1,31 @@
 // @ts-nocheck
-import React, { PureComponent } from 'react';
+import React, { FC } from 'react';
 import * as R from 'ramda';
 
-import techRadar from '../../../lib/zalando-tech-radar';
-import './radar.css';
+import { RadarContent } from './radarContent';
```

Comment:
I'm thinking about naming it 
`radar.container.tsx` where we could handle logic and use 
`radar.component.tsx` inside - which would be a dumb visual component. 

In radar container we could fetch contentful data and filter it and pass props to visual radar component. 
They both could be in the same directory called radar. 

What do you think?